### ISSUE TITLE
Option to disable native browser autocomplete - interrupts keyboard flow over elements

### ISSUE THREAD

**Issue by versedi** [trevoreyre/autocomplete#52]
In Chrome the native browser autocomplete pops up over the fetched results. This causes arrow down to select the native autocomplete suggestion instead of navigating over the results from this library.

Image should be self-explanatory:
![native-autocomplete-flow-off](https://user-images.githubusercontent.com/7837369/69718379-7cd88380-110e-11ea-8d6a-36e4b12319db.png)

This doesn't happen in any of your examples cause the inputs dont have any names assigned. 

Chrome: Version 77.0.3865.75 (Official Build) (64-bit)
Does not happen on Firefox.

Update
---

Uppon further debugging this is caused when the `input` has `name` that browser is able to cache in local storage.

See reproduced issue here on example:  https://codepen.io/versedi-ts/pen/eYYwqmx
Type something, press enter, refresh page and focus the input or type something in it.

**Comment by NesCafe62** [comment]
I tested, it also happends in firefox. The important fact to reproduce that you need to press Enter (so form will be submited) then reload the page and browser autocomplete appears if you start typing same value.
I probably know a workaround:
```
const autocomplete = new Autocomplete('#autocomplete', { /* ... */ });
autocomplete.input.setAttribute('autocomplete', 'off');
```

**Comment by versedi** [comment]
@NesCafe62 this indeed worked the way I would want it to.

Keeping my label and disabling default browser autocomplete.

**Comment by trevoreyre** [comment]
@versedi Chrome has been a bit of a moving target for this. Originally, I had the attribute `autocomplete="off"` set in the component, but Chrome was ignoring that, so it was updated [in this pull request](https://github.com/trevoreyre/autocomplete/pull/46). But I've since been hearing reports that it isn't working the other way now.

Do you happen to have any info or links to issues or blog posts on the current status of how to disable Chrome autofill? I'd be happy to accept a PR as well if you feel up to it.

**Comment by NesCafe62** [comment]
I tested in Chromium 77.0.3865.90 on LinuxMint 18.3 (x64) and Chrome 78.0.3904.108 on Windows 7 (x64): for both `autocomplete="off"` works as it should. Maybe older versions have different behavior. Another possible workaround is remove name attribute so browser couldn't remember values and have hidden input that changes value on selected (`submit` event).

**Comment by khalwat** [comment]
I can also verify that `autocomplete="off"` works as expected; currently Autocomplete-Vue is setting it to `autocomplete="disabled-autocomplete"` which doesn't work.

Google Chrome Version 79.0.3945.88 (Official Build) (64-bit)

ref: https://gist.github.com/niksumeiko/360164708c3b326bd1c8

**Comment by trevoreyre** [comment]
This should be fixed now in v2.0.4.
